src/SourceBuild/content/eng/Versions.props

```diff
@@ -32,5 +33,6 @@
     <MicrosoftExtensionsLoggingVersion>9.0.0-preview.2.24128.5</MicrosoftExtensionsLoggingVersion>
     <!-- command-line-api -->
     <SystemCommandLineVersion>2.0.0-beta4.24126.1</SystemCommandLineVersion>
+    <MicrosoftExtensionsFileSystemGlobbingVersion>8.0.0</MicrosoftExtensionsFileSystemGlobbingVersion>
```

Comment:
Please move this under the `runtime` section. I just noticed that other runtime packages already have a P2 version. We should be consistent so either upgrade this again back to P2 or downgrade the others to the 8.0.0 version.